web/crux/src/domain/agent.ts

```diff
@@ -184,7 +188,7 @@ export class Agent {
 
     this.eventChannel.next({
       id: this.id,
-      status: NodeConnectionStatus.UNREACHABLE,
+      status: 'unreachable',
```

Comment:
Using enums, way better thans strings, IMHO

Reply:
These are typed string literals. The enum was a generated by grpc.